src/Model/Item.php

```diff
@@ -170,6 +189,44 @@ public static function delete($item_id, $priority = PRIORITY_HIGH)
 		return true;
 	}
 
+	private static function deleteTagsFromItem($item)
```

Comment:
This function doesn't straight up delete tags, it merely updates the `tag` field from the `object` field. Please rename it to `updateTagsFromObject` or simply `updateTags`.

Reply:
It removes (deletes) tags from items that had been added with the corresponding item array that is now about to be deleted.

Reply:
Then I probably don't understand what tags are. I thought they were limited to a single item? From what you're saying, when we are deleting an item we a deleting tags from *other* items?

Reply:
People can add tags to the items of other people.

Reply:
Ah-ha! That's what I was missing.